docs/APIReference-Route.md

```diff
@@ -70,6 +76,72 @@ class ProfileRoute extends Relay.Route {
 }
 ```
 
+### prepareParams (static property)
+
+```
+static prepareParams: ?(prevParams: {[prevParam: string]: mixed}) => {[param: string]: mixed};
+```
+
+Routes can declare additional parameters, or conversion for parameters on initialisation. These can also be used as query parameters in your nested containers.
+
+#### Example
+
+```
+class ProfileRoute extends Relay.Route {
+  static queries = {
+    viewer: () => Relay.QL`query { viewer }`
+  };
+  static prepareParams = (prevParams) => {
+    // Make sure any params that ProfileRoute gets initialised with will still be passed down.
+    return {
+      ...prevParams,
+      limit: 10
+    }
+  }
+  // ...
+}
+
+const Child = Relay.createContainer((props) => ..., {
+  initialVariables: {
+    limit: null
+  },
+  fragments: {
+    viewer: ({limit}) => Relay.QL`
+      fragment on User {
+        friends(first: $limit) {
+          edges {
+            node {
+              name
+            }
+          }
+        }
+      }
+    `
+  }
+});
+
+const Container = Relay.createContainer((props) => (
+  <View>
+    <Child viewer={viewer} limit={this.props.relay.variables.limit} />
+    <Child viewer={viewer} limit={20} />
+  </View>
+  ), {
+  initialVariables: {
+    limit: null
+  },
+  fragments: {
+    viewer: ({limit}) => Relay.QL`
+      fragment on User {
+        name
+        ${Child.getFragment('viewer', {limit})}
+        ${Child.getFragment('viewer', {limit: 20})}
+      }
+    `
+  }
+});
+```
+In this example query parameters are being passed down to the `Child` component: One comes from the `ProfileRoute` via `prepareParams` and given `ProfileRoute` and `Container` are both being passed into `Relay.Renderer`, it automatically passed it down. The `Container` then passes two different `limit` variables down to the `Child` one from the `Route` and one from within.
+
```

Comment:
It's worth documenting this, but it doesn't really have to do with routes or prepareParams. What about adding a section to the RelayContainer guide (linked to from the Route guide) about variable passing?